kubeflow/jupyter/notebook_controller.libsonnet

```diff
@@ -23,6 +23,14 @@
           kind: "Notebook",
         },
       },
+      status: {
```

Comment:
Consider using subresource status and then calling UpdateStatus(): https://github.com/kubeflow/tf-operator/pull/939

This prevents bugs where the previous status is unexpectedly overwritten as a result of an update.

Reply:
we are using status subresource. see line 17